.github/workflows/lint-test.yml

```diff
@@ -31,8 +31,13 @@ jobs:
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
       - run: yarn --frozen-lockfile
       - run: yarn allow-scripts
-      # "Testing" (i.e. config validation) is also done in this step.
-      - run: yarn lint
+      - run: yarn lint # "Testing" (i.e. config validation) is also done here
+      - name: Validate RC changelog
+        if: ${{ startsWith(github.ref, 'release/') }}
+        run: yarn workspaces run auto-changelog validate --rc
```

Comment:
I don't think this will work in a monorepo. You'll need to pass in the path for each changelog.

Reply:
No it works, I've tried it locally. Similarly, `yarn workspaces run auto-changelog update` updates the changelog of every package.

Reply:
Oops, I was skimming this too quickly, missed the `workspaces` part. This makes sense now.